Tic_Tac_Toe.py

```diff
@@ -0,0 +1,65 @@
+def main():
+    print("=" * 30)
+    print("""Welcome to Tic Tac Toe
+GAME RULES:
+Each player can place one mark (or stone) per turn on the 3x3 grid
+The WINNER is who succeeds in placing three of their marks in a
+* horizontal,
+* vertical or
+* diagonal row
+Let's start the game...""")
+    board = [str(i + 1) for i in range(0, 9)]
+    player = "x"
+    print_board(board)
+    board = [" " for i in range(0, 9)]
+    while not evaulate_board(board,player):
+        if player == "x":
+            player = "o"
+        else:
+            player = "x"
+        change_board(board,player,move=get_move(player))
+        print_board(board)
+    print("=" * 40)
+    print(f"Congratulations, the player '{player}' WON!")
+
+
+def change_board(board,player,move):
+    while move > 8:
+        input("This position does't exist, press 'Enter' to try again...")
+        move = get_move(player)
+    while board[move] != " ":
+        input("This position is not free, press 'Enter' to try again...")
+        move = get_move(player)
+    board[move] = player
+
```

Comment:
You should have two blank lines between top-level functions https://pep8.org/#blank-lines